types.go

```diff
@@ -28,12 +31,16 @@ type ConfigResponse struct {
 }
 
 type ConfigRequest struct {
-	ClientVersion     string         `json:"client_version,omitempty"`
-	DeviceID          string         `json:"device_id,omitempty"`
-	SingboxVersion    string         `json:"singbox_version,omitempty"`
-	OS                string         `json:"os,omitempty"`
-	AppName           string         `json:"app_name,omitempty"`
-	PreferredLocation ServerLocation `json:"preferred_location,omitempty"`
-	UserID            string         `json:"user_id,omitempty"`
-	WGPublicKey       string         `json:"wg_public_key,omitempty"`
+	ClientVersion     string          `json:"client_version,omitempty"`
+	DeviceID          string          `json:"device_id,omitempty"`
+	RadianceVersion   string          `json:"radiance_version"`
+	SingboxVersion    string          `json:"singbox_version,omitempty"`
+	OS                string          `json:"os,omitempty"`
+	AppName           string          `json:"app_name,omitempty"`
+	PreferredLocation *ServerLocation `json:"preferred_location,omitempty"`
+	UserID            string          `json:"user_id,omitempty"`
+	ProToken          string          `json:"pro_token,omitempty"`
+	WGPublicKey       string          `json:"wg_public_key,omitempty"`
+	Country           string          `json:"country,omitempty"`
+	Ip                string          `json:"ip,omitempty"`
```

Comment:
Why do we need the client IP? I don't think we're actually using that on the server for anything, correct?

I also thought we don't need the pro token with our new auth. Is that right @reflog?

Reply:
@myleshorton if we want to constrain this API to be only for users that signed up - then yes, we can use the new auth method instead of the pro token

Reply:
Ah gotcha ok yeah i think we want it for all users.